shared/common-adapters/drag-and-drop.desktop.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+import * as React from 'react'
+import fs from 'fs'
+import * as Styles from '../styles'
+import {Box2} from './box'
+import Icon from './icon'
+import Text from './text'
+import type {Props} from './drag-and-drop'
+import logger from '../logger'
```

Comment:
just FYI a thing i've been doing lately is to put this on top
```
const Kb = {
Icon, Text, Box2
}
```

etc then you can do <Kb. whatever below
(you don't have to make this change in this pr but for future reference)